src/components/CioPlpBrowse/CioPlpBrowse.css

```diff
@@ -0,0 +1,77 @@
+.cio-plp-grid {
```

Comment:
I don't see this file imported to `styles.css` or the component itself. It's also got a lot of duplicate styles from the `CioPlpGrid` css, so I'm guessing that's where the `Browse` component its taking the styles from now, so this file is probably not necessary